portal-kernel/src/com/liferay/portal/kernel/settings/SettingsLocatorHelper.java

```diff
@@ -35,6 +35,8 @@ public Settings getCompanyPortletPreferencesSettings(
 	public Settings getConfigurationBeanSettings(
 		String configurationPid, Settings parentSettings);
 
+	public Settings getGlobalPortalPreferenceSettings(Settings parentSettings);
```

Comment:
I think this should be getSystemPreferenceSettings() for consistency with the terminology used across our documentation